src/ui/components/navigation/Navigation.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react';
+
+import { Link } from 'react-router-dom';
+
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Button from '@material-ui/core/Button';
+
+const Navigation: FunctionComponent = () => {
+  return (
+    <AppBar color="primary" position="static">
+      <Toolbar className="nav-container">
+        <Button color="inherit" component={Link} to="/">
```

Comment:
Those 3 buttons can be one reusable component which takes those attributes as props. We can use the component in here with a prop that receives array of objects eg: `[{name: ''home, to: '/'}, {name: ''login, to: '/login'}, {name: ''signup, to: '/signup'}]` and loop to display those. probably basic but can be useful when we expand links through out the pages